packages/storage-postgres/migrations/0072_deal_account_manager_set_null.sql

```
@@ -0,0 +1,3 @@
+ALTER TABLE "deals" DROP CONSTRAINT IF EXISTS "deals_account_manager_id_people_id_fk";
+--> statement-breakpoint
+ALTER TABLE "deals" ADD CONSTRAINT "deals_account_manager_id_people_id_fk" FOREIGN KEY ("account_manager_id") REFERENCES "public"."people"("id") ON DELETE set null ON UPDATE no action;
```

SQLFluff (4.2.2)

[error] ADD CONSTRAINT ... FOREIGN KEY should use NOT VALID to avoid locking the table while validating existing rows.

(PG01)